newdle/client/src/components/CreateNewdle.js

```diff
@@ -52,29 +43,104 @@ function TimeSlotsPage() {
   const dispatch = useDispatch();
   const participants = useSelector(getMeetingParticipants);
   return (
-    <div className={styles['time-slots-grid']}>
-      <Grid container>
-        <Grid.Row columns={2}>
-          <Grid.Column width={5}>
-            <Calendar />
-          </Grid.Column>
-          <Grid.Column width={11}>
-            <Availability participants={participants} />
-          </Grid.Column>
-        </Grid.Row>
-        <Grid.Row>
-          <div className={styles['button-row']}>
-            <Button color="violet" icon labelPosition="left" onClick={() => dispatch(setStep(1))}>
-              Back
-              <Icon name="caret left" />
+    <Grid container>
+      <Grid.Row columns={2}>
+        <Grid.Column width={5}>
+          <Calendar />
+        </Grid.Column>
+        <Grid.Column width={11}>
+          <Availability participants={participants} />
+        </Grid.Column>
+      </Grid.Row>
+      <Grid.Row>
+        <div className={styles['button-row']}>
+          <Button color="violet" icon labelPosition="left" onClick={() => dispatch(setStep(1))}>
+            Back
+            <Icon name="caret left" />
+          </Button>
+          <Button color="violet" icon labelPosition="right" onClick={() => dispatch(setStep(3))}>
+            Next step
+            <Icon name="caret right" />
+          </Button>
+        </div>
+      </Grid.Row>
+    </Grid>
+  );
+}
+
+async function createNewdle() {
+  const results = await client.createNewdle();
+  return results;
+}
+
+function FinalizePage() {
+  const [successMessage, setSuccessMessage] = useState(false);
```

Comment:
I'd go for `success` / `setSuccess` - in `somethingMessage` you usually expect an actual message, not a bool.